platforms/jvm/code-quality/src/main/java/org/gradle/api/plugins/quality/CheckstylePlugin.java

```diff
@@ -47,6 +47,8 @@ public abstract class CheckstylePlugin extends AbstractCodeQualityPlugin<Checkst
 
     public static final String DEFAULT_CHECKSTYLE_VERSION = "9.3";
     private static final String CONFIG_DIR_NAME = "config/checkstyle";
+
+    @SuppressWarnings("HidingField")
     private CheckstyleExtension extension;
```

Comment:
❔ What is this one about?

Reply:
These plugins hides `extension` field in super type. I changed the suppression of warning to errorprone plugin configuration like we do in other projects, e.g.:
```
errorprone {
    disabledChecks.addAll(
        "HidingField", // 3 occurrences
    )
}
```